draft-ietf-quic-transport.md

```diff
@@ -1854,6 +1902,49 @@ alive when a stream is open. The default is to send a PING frame after 15
 seconds of quiescence. A PING frame has no additional fields.
 
 
+## NEW_CONNECTION_ID Frame {#frame-new-connection-id}
+
+A server sends a NEW_CONNECTION_ID to provide the client with alternative
+connection IDs that can be used to break linkability when migrating connections
+(see {{migration-linkability}}).
+
+The NEW_CONNECTION_ID is as follows:
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|       Sequence (16)           |
```

Comment:
Yes, this was part of the reasoning, the other was that at 255 it's much harder to get the wrapping right.  Of course, if we decided to do that we really would run out of packet numbers.  The fix, make the packet number much bigger, the 64-bit limit is pretty arbitrary anyway and the nonce for most AEADs is longer than that, meaning that we could go to as much as 96-bit without a redesign.